bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -751,10 +751,10 @@
 				<option value="opening">Opening</option>
 				<option value="stop">Stop</option>
 				<option value="closing">Closing</option>
-				<option value="limSwitch_open">Limit Switch Open</option>
-				<option value="limSwitch_close">Limit Switch Close</option>
-				<option value="state_open">Opened</option>
-				<option value="state_close">Closed</option>
+				<option value="limSwitchOpen">Limit Switch Open</option>
+				<option value="limSwitchOlose">Limit Switch Close</option>
```

Comment:
Is that a typo? limSwitchOlose is it limSwitchClose?

Reply:
ops ... fixed ! (was Close)